### DISCUSSION TITLE
Support for native deno package

### DISCUSSION THREAD

**Discussion by sant123** [supabase/supabase#13780] [Feature Requests]
Hi, currently Supabase is not available officially in https://deno.land/x. The current docs says to install it through `esm`, however would be better for Deno users to install it through deno land.

Some of the benefits:

+ Docs can be read in deno.land for each method, field, interface, type, etc.
+ Support for immutable versions, sometimes https://esm.sh brings different code with same versioning causing deno.lock to fail
+ Can be available to other JS runtimes that support import modules by URL out of the box
+ No bundling process, the code will be served as it was published

Please let me know what you think, thanks!

**Comment by Evertt** [comment]
Yes please! 🙏
